etc/lime-elements.api.md

```diff
@@ -1000,7 +999,6 @@ namespace JSX_2 {
         // @beta
         "type"?: ChipType;
     }
-    // (undocumented)
     interface LimelChipSet {

```

Comment:
@jgroth I have no idea why api-extractor suddenly thinks this is any less documented than it was before… 🤷‍♂️

The change means I'll need an approval from you though 😅

Reply:
The rest of the PR has been approved by Per, so I'm going to use my admin privileges to bypass the need for another maintainer than myself to review 😊